### PR TITLE
Soaring updates

### PR DESCRIPTION
**Pull request by samuelctabor**
Update for new soaring behaviours described [here](https://discuss.ardupilot.org/t/soaring-gliding-for-planes/15323/250?u=samuel_tabor) based on top of [existing PR for soaring RC option](https://github.com/ArduPilot/ardupilot_wiki/pull/2914/files).

### CODE REVIEW

**Comment by Hwurzburg** [comment]
Since this duplicates everything in #2914 and expands on it, we can close that one, and change the reference link in Upcoming features to this  PR....

**Comment by samuelctabor** [comment]
> Since this duplicates everything in #2914 and expands on it, we can close that one, and change the reference link in Upcoming features to this PR....

Done, thanks Henry.

File: plane/source/docs/soaring.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ This picture shows the different phases of flight when using the soaring
...
+#. If the aircraft descends to :ref:`SOAR_ALT_MIN<SOAR_ALT_MIN>` altitude, throttle is re-enabled and the aircraft will begin to climb to the altitude of the next waypoint. If that waypoint altitude is less than :ref:`SOAR_ALT_CUTOFF<SOAR_ALT_CUTOFF>` altitude, then Soaring can not begin before reaching the waypoint. If it is above :ref:`SOAR_ALT_CUTOFF<SOAR_ALT_CUTOFF>` altitude, then Soaring can occur once that altitude has been reached.
```
Comment (Hwurzburg) [inline plane/source/docs/soaring.rst:23]: suggested change:
...the aircraft will begin to climb to the altitude of the next waypoint, in AUTO, or to :ref:`SOAR_ALT_CUTOFF<SOAR_ALT_CUTOFF>` altitude in CRUISE or FBWB.

Comment (samuelctabor) [reply plane/source/docs/soaring.rst]: Thanks, updated.

**Comment by samuelctabor** [comment]
Closing in favour of #3336

**Pull request closed without merging**